gradio/blocks.py

```diff
@@ -516,6 +517,7 @@ def __init__(
             js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
             head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
             fill_height: Whether to vertically expand top-level child components to the height of the window. If True, expansion occurs when the scale value of the child components >= 1.
+            delete_cache: A tuple corresponding [frequency, age] both expressed in number of seconds. Every `frequency` seconds, the contents of GRADIO_CACHE_DIR created by this Blocks instance will be deleted if more than `age` seconds have passed since the file was created. The cache will be deleted entirely when the server restarts. If None, no cache deletion will occur.
```

Comment:
```suggestion
            delete_cache: A tuple corresponding [frequency, age] both expressed in number of seconds. Every `frequency` seconds, the temporary files created by this Blocks instance will be deleted if more than `age` seconds have passed since the file was created. The cache will be deleted entirely when the server restarts. If None, no cache deletion will occur.
```

Reply:
I'm a little concerned that users might configure this parameter incorrectly, which could lead to undesired behavior since its not clear what a good default value is. For example, setting it to `(60, 60)` would probably break many demos since temporary files will be deleted before their turn in the queue. 

Some ideas:

1. Provide some guidance on a good value in the docstring, e.g. "e.g. setting this to (86400, 86400)" will delete temporary files every day.

2. Allow users to just set this to `True` which will  still do this: "The cache will be deleted entirely when the server restarts." but not have the background task. Should be much easier for users to configure.

Reply:
Can you explain 2 a bit more? How would removing the background task address this comment:

> I think we should provide a mechanism to delete files that does not require a server restart. E.g. all files older than a certain timestamp get deleted